ingress/dashboard_ingress_setup.md

```diff
@@ -0,0 +1,216 @@
+```
+
+3. Add Let's Encypt issuers
+
+```bash
+kubectl apply -f issuers.yaml
+```
+
+::: code-group
+```yaml [issuers.yaml]
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: letsencrypt-prod
+  namespace: koor-ceph
+spec:
+  acme:
+    # The ACME server URL
+    server: https://acme-v02.api.letsencrypt.org/directory
+    # Email address used for ACME registration
+    email: admin@koor.tech
+    # Name of a secret used to store the ACME account private key
+    privateKeySecretRef:
+      name: letsencrypt-prod
+    # Enable the HTTP-01 challenge provider
+    solvers:
+      - http01:
+          ingress:
+            ingressClassName: nginx
+```
+:::
+
+References: 
+1. https://cert-manager.io/docs/installation/helm/
+2. https://cert-manager.io/docs/tutorials/getting-started-with-cert-manager-on-google-kubernetes-engine-using-lets-encrypt-for-ingress-ssl/
+
+## Install Nginx Ingress and a Load Balancer
+
+1. Create a Load Balancer in Hetzner. Choose the network of the cluster. 
+Remove all targets and services since they will be added automatically. 
+Name your load balancer `demolb`
+
+2. Install ingress-nginx using the helm chart
+
+```bash
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm repo update
+helm -n ingress-nginx install ingress-nginx ingress-nginx/ingress-nginx --create-namespace -f nginx-values.yaml
+```
+
+::: code-group
+```yaml [nginx-values.yaml]
+controller:
+  config:
+    entries:
+        use-forwarded-headers: "true"
+        compute-full-forwarded-for: "true"
+        use-proxy-protocol: "true"
+  service:
+    annotations:
+      # This is the name of the load balancer just created
+      load-balancer.hetzner.cloud/name: "demolb"
+```
+:::
+
+References:
+1. https://community.hetzner.com/tutorials/howto-k8s-authentication-with-load-balancer
+2. https://kubernetes.github.io/ingress-nginx/deploy/#ovhcloud
+
+## Install Koor Data Control Center and Enable Ingress
+Finally we are ready to install KDCC using [the helm chart](https://github.com/koor-tech/data-control-center/tree/main/charts/data-control-center).
+
+```bash
+helm repo add data-control-center https://koor-tech.github.io/data-control-center
+helm repo update
+helm install --create-namespace --namespace rook-ceph data-control-center data-control-center/data-control-center -f values.yaml
+```
+
+::: code-group
+```yaml [values.yaml]
+config:
+  users:
+    - username: admin
+      password: "<SomeStrongPassword>"
+```
+:::
+
+## Create a DNS entry for the demo system
+
+Add the following to the `infra` repository, we add names we will be using as
+CNAMES, the ngnix ingress will handle routing the traffic based on subdomain
+name. See more (here)[https://kubernetes.io/docs/concepts/services-networking/ingress/#name-based-virtual-hosting]
+
+```js
+  A("k8s-lb", "<Load balancer IPv4>"),
+  AAAA("k8s-lb", "<Load balancer IPv6>"),
+  CNAME("demo", "k8s-lb"),
+  CNAME("ceph", "k8s-lb"),
+```
+
+## Setup 
+
+::: code-group
```

Comment:
ok, so we're using an explicit ingress instead of specifying it in the helm chart?

Reply:
yep, is there any downside to this, I can update docs if there is, thanks!

Reply:
No downsides, just an observation.